tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml

```diff
@@ -0,0 +1,13 @@
+---
+__FORKDELTA_CUSTOM_SYMBOL: AVOID CONFLICT WITH 0xd9a0658b7cc9ec0c57e8b20c0920d08f17e747be
+addr: '0xc56b13ebbcffa67cfb7979b900b736b3fb480d78'
+decimals: 8
+description: >-
+  Decentralized social network
+  You take back power with every social connection
+links:
+- Website: https://sphere.social/hello/
+- Google Play: https://play.google.com/store/apps/details?id=com.sphere.social
+- Whitepaper: https://sphere.social/wp-content/uploads/2017/12/Sphere_Whitepaper_v1.7.4.pdf
+name: Social Activity Token
+symbol: SATO
```

Comment:
> 4) Non-critical issue: expected symbol to be SAT, but got SATO
        in test_symbol_equals_erc20_symbol: name should reflect the symbol defined by the contract

@therealMikkoK Looks like we replaced "SAT" with "SATO" in https://github.com/forkdelta/tokenbase/commit/70cd2f48d0976f25c674db9708330a2bb4e48781. Let's use `symbol: SAT` and put the custom "SATO" in `__FORKDELTA_CUSTOM_SYMBOL`: `__FORKDELTA_CUSTOM_SYMBOL: SATO`.

Reply:
Need assistance on this @freeatnet